app/helpers/application_helper.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def sortable(column)
+    direction = column == params[:sort] && params[:direction] == "asc" ? "desc" : "asc"
```

Comment:
shouldn't the conditional check be `params[:direction] == "asc"? "asc": "desc"` instead?